ecosystem.html

```diff
@@ -0,0 +1,1629 @@
+                    class="btn btn-outline-black"
+                    >Visit the documentation</a
+                  >
+                </div>
+              </div>
+            </a>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-6 col-lg-4">
+            <a href="https://receptor.readthedocs.io/en/latest/index.html">
+              <div class="card border">
+                <div class="card-body">
+                  <h4 class="card-title">Receptor</h4>
+                  <p class="card-text">
+                    Receptor is an overlay network that distributes workloads
+                    across a large and dispersed collection of nodes.
+                  </p>
+                </div>
+                <div class="card-footer">
+                  <a
+                    href="https://receptor.readthedocs.io/en/latest/index.html"
+                    class="btn btn-outline-black"
+                    >Visit the documentation</a
+                  >
+                </div>
+              </div>
+            </a>
+          </div>
+          <div class="col-md-6 col-lg-4">
+            <a href="https://ansible.github.io/ansible-sign/">
+              <div class="card border">
+                <div class="card-body">
+                  <h4 class="card-title">Sign</h4>
+                  <p class="card-text">
+                    Sign is a utility for signing and verifying Ansible content.
+                  </p>
+                </div>
+                <div class="card-footer">
+                  <a
+                    href="https://ansible.github.io/ansible-sign/"
+                    class="btn btn-outline-black"
+                    >Visit the documentation</a
+                  >
+                </div>
+              </div>
+            </a>
+          </div>
+          <div class="col-md-6 col-lg-4">
+            <a href="https://galaxyng.netlify.app/">
+              <div class="card border">
+                <div class="card-body">
+                  <h4 class="card-title">Galaxy NG</h4>
+                  <p class="card-text">
+                    Galaxy NG helps you jump-start your automation project with
+                    great content from the Ansible community.
+                  </p>
+                </div>
+                <div class="card-footer">
+                  <a
+                    href="https://galaxyng.netlify.app/"
+                    class="btn btn-outline-black"
+                    >Visit the documentation</a
+                  >
+                </div>
+              </div>
+            </a>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-6 col-lg-4">
+            <a href="https://ansible-rulebook.readthedocs.io/en/latest/">
+              <div class="card border">
+                <div class="card-body">
+                  <h4 class="card-title">Rulebook</h4>
```

Comment:
Ditto on the name. Maybe we as a team should just decide what we want to call these. I can understand the desire to keep the name simple for sure, but we should also keep it consistent with the project docs themselves.